src/components/route-home/category/CategoryRadioFieldset.tsx

```diff
@@ -0,0 +1,86 @@
+import { ChangeEvent, Dispatch, FC, InputHTMLAttributes, SetStateAction, useId } from 'react';
+import styled from '@emotion/styled';
+
+import { categories, CategoryType } from './type';
+
+const RADIO_CATEGORY_NAME = 'category';
+
+interface Props {
+  currentCategory: CategoryType;
+  setCurrentCategory: Dispatch<SetStateAction<CategoryType>>;
+}
+
+const CategoryRadioFieldset: FC<Props> = ({ currentCategory, setCurrentCategory }) => {
+  const onChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.currentTarget;
+    if (value === '일상' || value === '여행' || value === '운동') {
```

Comment:
src/components/route-home/category/type.ts 여기에 정의된거 가지고와서
이렇게 바꾸면 좋을 것 같아요
```suggestion
    if (categories.include(value)) {
```

Reply:
해야지 했었던 건데 놓쳤었네요 👍 👍 👍

Reply:
`includes`를 통과한 이후에 타입 추론이 안되네요 😢 

```ts
const arr = ['a', 'b', 'c'] as const;
const foo: string = 'z';

if (arr.includes(foo)) {
  // foo가 아직 string 타입으로 추론됨
}
```

`isIn`이라는 유틸 간단히 만들어서 사용했어요 !

```ts
export const isIn = <T>(values: ReadonlyArray<T>, x: unknown): x is T => {
  return values.includes(x as T);
};

const arr = ['a', 'b', 'c'] as const;
const foo: string = 'z';

if (isIn(arr, foo)) {
  // foo가 'a' | 'b' | 'c'로 추론됨
}
```